tests/unit/jobserver/test_honeycomb.py

```diff
@@ -36,3 +39,13 @@ def test_format_trace_link():
     url = format_trace_link(job)
     assert "trace_start_ts=1655297940&trace_end_ts=1655298060" in url
     assert "bennett-institute-for-applied-data-science" in url
+
+
+@pytest.mark.freeze_time("2022-10-12 17:00")
+def test_format_jobrequest_concurrency_link():
+    job_request = JobRequestFactory(identifier="jpbaeldzjqqiaolg")
+
+    url = format_jobrequest_concurrency_link(job_request)
+    expected_url = "https://ui.honeycomb.io/bennett-institute-for-applied-data-science/environments/production/datasets/jobrunner?query=%7B%22start_time%22%3A1665593940%2C%22end_time%22%3A1665594000%2C%22granularity%22%3A0%2C%22breakdowns%22%3A%5B%22name%22%5D%2C%22calculations%22%3A%5B%7B%22op%22%3A%22CONCURRENCY%22%7D%5D%2C%22filters%22%3A%5B%7B%22column%22%3A%22enter_state%22%2C%22op%22%3A%22!%3D%22%2C%22value%22%3A%22true%22%7D%2C%7B%22column%22%3A%22name%22%2C%22op%22%3A%22!%3D%22%2C%22value%22%3A%22RUN%22%7D%2C%7B%22column%22%3A%22name%22%2C%22op%22%3A%22!%3D%22%2C%22value%22%3A%22JOB%22%7D%2C%7B%22column%22%3A%22name%22%2C%22op%22%3A%22!%3D%22%2C%22value%22%3A%22job%22%7D%2C%7B%22column%22%3A%22job_request%22%2C%22op%22%3A%22%3D%22%2C%22value%22%3A%22jpbaeldzjqqiaolg%22%7D%5D%2C%22filter_combination%22%3A%22AND%22%2C%22orders%22%3A%5B%7B%22op%22%3A%22CONCURRENCY%22%2C%22order%22%3A%22descending%22%7D%5D%2C%22havings%22%3A%5B%5D%2C%22limit%22%3A1000%7D"
```

Comment:
Why not define this unquoted? It'll be easier to make changes to it in future?

And assert directly against it below?

Reply:
ah, yeah, this started life as a template link generated by the honeycomb interface itself, so it's in the format that they generate/specify. But I agree it would be easier to work with unquoted. hmmm